src/sources/soundsourceffmpeg.h

```diff
@@ -304,5 +192,3 @@ class SoundSourceProviderFFmpeg : public SoundSourceProvider {
 };
 
 } // namespace mixxx
```

Comment:
I kinda understand that everything is in one file and there is just Inner classes but it makes understanding this class very very difficult.